manifests/configaware.go

```diff
@@ -195,13 +172,13 @@ func (frs *fileResourceStore) setConfigFileWorkloadContainerImage(ctx context.Co
 	return nil
 }
 
-func (frs *fileResourceStore) updatePatchFile(ctx context.Context, cf *ConfigFile,
+func (ca *configAware) updatePatchFile(ctx context.Context, cf *ConfigFile,
 	updateF func(previousManifests []byte) ([]byte, error)) error {
 
 	patchUpdated := *cf.PatchUpdated
-	generatedManifests, patchedManifests, patchFilePath, err := frs.getGeneratedAndPatchedManifests(ctx, cf, patchUpdated)
+	generatedManifests, patchedManifests, patchFilePath, err := ca.getGeneratedAndPatchedManifests(ctx, cf, patchUpdated)
 	if err != nil {
-		relConfigFilePath, err := filepath.Rel(frs.baseDir, cf.Path)
+		relConfigFilePath, err := filepath.Rel(ca.rawfiles.baseDir, cf.Path)
```

Comment:
I don't like that we piggyback on `rawFiles`'s fields for functionality unrelated to `rawFiles`. I would keep a copy.